js/test/components/operatorSignIn/attestation.test.tsx

```diff
@@ -216,4 +227,42 @@ describe("Attestation", () => {
     expect(onComplete).not.toHaveBeenCalled();
     jest.useRealTimers();
   });
+
+  describe("expiry", () => {
+    test("displays expired mode if expired", () => {
+      const view = render(
+        <Attestation
+          badge="123"
+          prefill={false}
+          onComplete={jest.fn()}
+          loading={false}
+          employees={EMPLOYEES}
+          certifications={CERTIFICATIONS_ONE_EXPIRED}
+        />,
+      );
+      expect(view.getByText("Expired card")).toBeInTheDocument();
+      expect(
+        view.getByText("Continue to Fit for Duty Check."),
+      ).toBeInTheDocument();
+    });
+
+    test("can bypass expired mode", async () => {
+      const view = render(
+        <Attestation
+          badge="123"
+          prefill={false}
+          onComplete={jest.fn()}
+          loading={false}
+          employees={EMPLOYEES}
+          certifications={CERTIFICATIONS_ONE_EXPIRED}
+        />,
+      );
+      await userEvent.click(
+        view.getByRole("button", { name: "Continue to Fit for Duty Check →" }),
+      );
+      expect(
+        view.getByRole("button", { name: "Complete Fit for Duty Check" }),
+      ).toBeInTheDocument();
+    });
+  });
```

Comment:
these tests are redundant. You don't need to assert the existence of the button and then assert that you can click it, just one test that asserts clicking the button would work.

I suggest deleting the first test, and copying the one unique assertion from that test (`getByText("ExpiredCard")`) into the second.
```suggestion
    test("can bypass expired mode", async () => {
      const view = render(
        <Attestation
          badge="123"
          prefill={false}
          onComplete={jest.fn()}
          loading={false}
          employees={EMPLOYEES}
          certifications={CERTIFICATIONS_ONE_EXPIRED}
        />,
      );
      expect(view.getByText("Expired card")).toBeInTheDocument();
      await userEvent.click(
        view.getByRole("button", { name: "Continue to Fit for Duty Check →" }),
      );
      expect(
        view.getByRole("button", { name: "Complete Fit for Duty Check" }),
      ).toBeInTheDocument();
    });
  });
```

Reply:
fixed 👍